### PR TITLE
Update announcements.md

### PR DESCRIPTION
**Pull request by toco-cam**
## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [ ] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/reference/announcements.md [hunk 1]
```diff
@@ -4,6 +4,24 @@ title: "Announcements"
 description: "Important announcements including deprecation & removal notices"
 ---
 
+
+## Camunda 8.5
+
+
+### Support for Amazon OpenSearch
+As of the 8.5 release, Optimize is now also compatible with [Amazon OpenSearch](https://aws.amazon.com/de/opensearch-service/) 2.5.x. Note that using Amazon OpenSearch requires [setting up a new Camunda installation](/self-managed/platform-deployment/overview.md). A migration from previous versions or Elasticsearch environments is currently not supported.
```
Comment (grlimacan) [inline docs/reference/announcements.md:12]:
```suggestion
With the 8.5 release, Optimize is now also compatible with [Amazon OpenSearch](https://aws.amazon.com/de/opensearch-service/) 2.5+. Note that using Amazon OpenSearch requires [setting up a new Camunda installation](/self-managed/platform-deployment/overview.md). A migration from previous versions or Elasticsearch environments is currently not supported.
```

File: docs/reference/announcements.md [hunk 2]
```diff
@@ -4,6 +4,24 @@ title: "Announcements"
...
+
+### Known limitations
+
+This release contains the following limitations:
+
+- In **Optimize `8.5.0`**
+  - **Limitation**
+    - **Description:** OpenSearch support in Optimize is limited to data import and the raw data report.
+    - **Reference:** n/a
+    - **Mitigation:** Optimize can be installed and used in production with limited reporting functionality. Optimize imports all process data generated by Zebee. All repoting functionality as described in the docs will be delivered with upcoming patches.
```
Comment (grlimacan) [inline docs/reference/announcements.md:22]:
```suggestion
    - **Mitigation:** Optimize can be installed and used in production with limited reporting functionality. Optimize imports all process data generated by Zebee. All reporting functionality as described in the docs will be delivered with upcoming patches.
```

File: docs/reference/announcements.md (hunk 1 above)
Comment (grlimacan) [inline docs/reference/announcements.md:12]: Do we want to mention that upgrade from older OpenSearch versions (i.e. 1.x) is currently not supported but will be supported in the future?

File: docs/reference/announcements.md [hunk 3]
```diff
@@ -4,6 +4,24 @@ title: "Announcements"
 description: "Important announcements including deprecation & removal notices"
 ---
 
+
+## Camunda 8.5
+
+
+### Support for Amazon OpenSearch
+As of the 8.5 release, Optimize is now also compatible with [Amazon OpenSearch](https://aws.amazon.com/de/opensearch-service/) 2.5.x. Note that using Amazon OpenSearch requires [setting up a new Camunda installation](/self-managed/platform-deployment/overview.md). A migration from previous versions or Elasticsearch environments is currently not supported.
+
```
Comment (grlimacan) [inline docs/reference/announcements.md:13]: Probably out of scope here, but the page you're linking to makes reference that Operate, Tasklist and Optimize require ElasticSearch. This should be updated to ElasticSearch/OpenSearch

**Review by grlimacan — APPROVED** [review]
Looks good generally, just some comments. Pre-approving

File: docs/reference/announcements.md (hunk 1 above)
Comment (toco-cam) [reply docs/reference/announcements.md]: @grlimacan, why will it be supported in the future? We are only aware of one customer using OpenSearch with Camunda 8, and we have found a customer solution for him.

Comment (grlimacan) [reply docs/reference/announcements.md]: All right, then we don't need to worry about it. @RomanJRW I remember the request came from you that we should support the upgrade, was it about this customer? Is it still something we need to implement in the future or can we descope it completely?

Comment (RomanJRW) [reply docs/reference/announcements.md]: There are two things to consider:
- Upgrading from old versions of OpenSearch: We don't care about supporting this, we never do, even for ES. Customers manage their own DBs, we just say what is compatible
- Upgrading from Optimize 3.7 -> 3.13.x: We care about this. Customers are running 3.7 still with an old OS DB, and cannot move to newer Optimize versions until we offer official support. When we do, we need to provide an upgrade script that takes then from 3.7 schema to whatever schema they are on now

File: docs/reference/announcements.md [hunk 4]
```diff
@@ -4,6 +4,22 @@ title: "Announcements"
...
+### Known limitations
+
+This release contains the following limitations:
+
+- In **Optimize `8.5.0`**
+  - **Limitation**
+    - **Description:** OpenSearch support in Optimize is limited to data import and the raw data report.
+    - **Reference:** n/a
+    - **Mitigation:** Optimize can be installed and used in production with limited reporting functionality. Optimize imports all process data generated by Zebee. All reporting functionality as described in the docs will be delivered with upcoming patches.
```
Comment (grlimacan) [inline docs/reference/announcements.md:21]:
```suggestion
    - **Mitigation:** Optimize can be installed and used in production with limited reporting functionality. Optimize imports all process data generated by Zeebe. All reporting functionality as described in the docs will be delivered with upcoming patches.
```

File: docs/reference/announcements.md [hunk 5]
```diff
@@ -4,6 +4,22 @@ title: "Announcements"
 description: "Important announcements including deprecation & removal notices"
 ---
 
+## Camunda 8.5
+
+### Support for Amazon OpenSearch
+
+With the 8.5 release, Optimize is now also compatible with [Amazon OpenSearch](https://aws.amazon.com/de/opensearch-service/) 2.5+. Note that using Amazon OpenSearch requires [setting up a new Camunda installation](/self-managed/platform-deployment/overview.md). A migration from previous versions or Elasticsearch environments is currently not supported.
```
Comment (grlimacan) [inline docs/reference/announcements.md:11]: This text suggests that a migration from Elasticsearch environments might be eventually supported, which we don't intend to do.

**Review by grlimacan — DISMISSED** [review]
Pre-approving, just 2 small things

File: docs/reference/announcements.md (hunk 3 above)
Comment (christinaausley) [reply docs/reference/announcements.md:12]: Are we still looking to clean this up?

Comment (christinaausley) [reply docs/reference/announcements.md:12]: @toco-cam Are you still looking for someone to update here? Not sure if Amara got to it before she headed out.

**Comment by christinaausley** [comment]
@toco-cam Let me know when you're ready for final review following a few open comments 😄

File: docs/reference/announcements.md (hunk 3 above)
Comment (toco-cam) [reply docs/reference/announcements.md:12]: @christinaausley if @akeller did not do yet, yes.

File: docs/reference/announcements.md (hunk 5 above)
Comment (toco-cam) [reply docs/reference/announcements.md]: @grlimacan I changed the text to:
"A migration from previous versions or Elasticsearch environments is not supported."
Does this make it more clear?

Comment (grlimacan) [reply docs/reference/announcements.md]: Sounds good

**Review by christinaausley — APPROVED** [review]

File: docs/reference/announcements.md (hunk 3 above)
Comment (christinaausley) [reply docs/reference/announcements.md:12]: Merging this PR in, but let me know if I can assist here with a follow up PR @akeller 👍

Comment (akeller) [reply docs/reference/announcements.md:12]: This fell off my radar. I will create an issue and PR this week.

Comment (akeller) [reply docs/reference/announcements.md:12]: Ah! This is no longer relevant. This page was reworked and the dependencies were moved to the Supported Environments page. That's why I didn't do it 🙂 Resolving this conversation

**Pull request merged**